content/accessible/use-semantic-html/index.md

```diff
@@ -84,7 +85,7 @@ and `SPACE` to attempt to click on them.
 </div>
```

Comment:
The Glitch embed above is nice, but as a user, I'm more interested in seeing the code differences between the two (rather than just a button that works and a button that doesn't).  Or maybe add more explicit instructions to also try clicking them manually to see that the div button actually does work in some ways?